src/data_structure/field2D.java

```diff
@@ -0,0 +1,33 @@
+package data_structure;
+
+public class field2D { // we need that class to gather information about rows and column in one fieldWritable
+
+    private fieldInfo column;
+    private fieldInfo row;
+    private fieldWritable writable;
+    private Type type;
+
+    public fieldInfo getColumn() {
```

Comment:
I think, we need a constructor, setter or default values to can fill this fields.